src/app.ts

```diff
@@ -1,27 +1,37 @@
 import express from 'express';
 import bodyParser from 'body-parser';
+import passport from 'passport';
+import cors from 'cors';
 
-import sequelize from './utils/database';
 import routes from './routes/user-routes';
 import groupRouter from './routes/group-routes';
 import groupUsersRouter from './routes/group-users-routes';
+import authRouter from './routes/auth-routes';
+
 import { isOperationalError, logError, returnError } from './utils/handleError/helpers';
+import sequelize from './utils/database';
 import HttpError from './utils/handleError/httpError';
 import httpLogger from './utils/handleError/httpLogger';
 import logger from './utils/handleError/logger';
+import jwtPassport from './middleware/passport';
 
 
 const app = express();
 const PORT = process.env.PORT || 3000;
 
+app.use(passport.initialize());
```

Comment:
As I can see you don't use the passport. Why do you need it?

Reply:
yes you right 👍